includes/widgets/image.php

```diff
@@ -447,11 +458,25 @@ protected function register_controls() {
 			]
 		);
 
+		$this->add_group_control(
+			Group_Control_Box_Shadow::get_type(),
+			[
+				'name' => 'image_box_shadow_hover',
+				'exclude' => [
+					'box_shadow_position',
+				],
+				'selector' => '{{WRAPPER}}:hover img',
+			]
+		);
+
 		$this->add_control(
 			'background_hover_transition',
 			[
 				'label' => esc_html__( 'Transition Duration', 'elementor' ),
 				'type' => Controls_Manager::SLIDER,
+				'default' => [
```

Comment:
It will affect existing pages, is it OK?

Reply:
Maybe we don't need this default value, especially if it affects existing pages. In any case, we have a default fallback in a CSS variable.

Reply:
@rami-elementor Default control removed